tests/jerry/arithmetics-bignums.js

```diff
@@ -23,8 +23,18 @@ assert(big == 2147483648); // overflow on 32bit numbers
 big++;
 assert(big == 2147483649); // overflow on 32bit numbers
 
-assert ((1152921504606846976).toString() === "1152921504606847000")
+assert ((1152921504606846976).toString() === "1152921504606847000");
 
 assert (1.797693134862315808e+308 === Infinity);
 
 assert (9999999999999999 == 10000000000000000);
+
+assert(9007199254740993 == 9007199254740992);
+
+assert(9007199254740992 == 9007199254740992);
+
+assert(9007199254740994 == 9007199254740994);
+
```

Comment:
I would also add asserts for the following:
```
9007199254740995 -> 9007199254740996
18014398509481989 -> 18014398509481988
18014398509481990 -> 18014398509481992
18014398509481991 -> 18014398509481992
18014398509481993 -> 18014398509481992
18014398509481994 -> 18014398509481996
```
I think these would cover all of the possible cases.